src/components/ExportLicenseAsCSVModal/ExportLicenseAsCSVModal.js

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { mapValues, pickBy } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  Checkbox,
+  Layout,
+  Modal,
+  ModalFooter,
+} from '@folio/stripes/components';
+
+import css from './ExportLicenseAsCSVModal.css';
+
+export default class ExportLicenseAsCSVModal extends React.Component {
+  static propTypes = {
+    onCompareLicenseTerms: PropTypes.func,
+    onClose: PropTypes.func,
+    selectedLicenses: PropTypes.arrayOf(PropTypes.string),
+    terms: PropTypes.arrayOf(PropTypes.object),
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.licenseInformation = {
+      name: <FormattedMessage id="ui-licenses.prop.name" />,
```

Comment:
Heads up: you should never expect an object to keep the ordering of the keys that you lexically set them as, that's not part of the Object spec. So since you use this object to render a list, if you always want the list to always be in order name -> startDate -> endDate etc, this (and I guess the terms stuff as well) should be an array.

Reply:
Ah yes